frontend/magic_links/routes.py

```diff
@@ -54,8 +64,8 @@ def new():
     users email address.
     """
     # Default to COF while we only have one fund
-    fund_id = request.args.get("fund_id", Config.FUND_ID_COF)
-    round_id = request.args.get("round_id")
+    fund_id = request.args.get("fund_id", Config.COF_FUND_ID)
```

Comment:
Should these use the defaults defined in config?
```suggestion
    fund_id = request.args.get("fund_id", Config. DEFAULT_FUND_ID)
```